cogs/qrchecker/commandsCore.py

```diff
@@ -19,3 +23,47 @@ async def cmdQrCheckerToggle(self, ctx: Context):
         else:
             await guildConfig.get_attr(KEY_ENABLED).set(True)
             await ctx.send("QR code checking is now **enabled** for this guild.")
+
+    async def cmdQrCheckerShow(self, ctx: Context):
+        """Show current settings"""
+        pixels = await self.config.get_attr(KEY_MAX_IMAGE_PIXELS)()
+        enabled = await self.config.guild(ctx.guild).get_attr(KEY_ENABLED)()
+        pixelsStr = f"{pixels or 'Unlimited'} pixels"
+        enabledStr = "Yes" if enabled else "No"
+        globalSettingsStr = "\n".join(
+            (
+                "**__Global settings__**",
+                f"Max pixels: **{pixelsStr}**",
+            )
+        )
+        guildSettingsStr = "\n".join(
+            (
+                "**__Guild settings__**",
+                f"Enabled: **{enabledStr}**",
+            )
+        )
+        msg = "\n\n".join((globalSettingsStr, guildSettingsStr))
+
+        await ctx.send(msg)
+
+    async def cmdQrCheckerMaxPixels(self, ctx: Context, *, pixels: Optional[int]):
+        """Set the maximum image pixels to check.
+
+        Binary images are loaded into RAM in Pillow, which uses RAM. If your bot is
+        running on a system with limited RAM, set this to a low value to avoid OOM
+        killer from killing your bot.
+
+        Parameters
+        ----------
+        pixels: Optional[int]
+            The maximum number of pixels in an image to check.
+        """
+        if pixels and pixels < 0:
+            await ctx.send("Please enter a positive number")
```

Comment:
Micronit: since the other sentences have ending marks, might as well do an exclamation mark for this one.

```suggestion
            await ctx.send("Please enter a positive number!")
```